src/mach/mod.rs

```diff
@@ -147,7 +147,12 @@ impl<'a> MachO<'a> {
         }
     }
     /// Parses the Mach-o binary from `bytes` at `offset`
-    pub fn parse(bytes: &'a [u8], mut offset: usize) -> error::Result<MachO<'a>> {
+    pub fn parse(bytes: &'a [u8], offset: usize) -> error::Result<MachO<'a>> {
+        Self::parse_2(bytes, offset, false)
+    }
+
+    /// Parses the Mach-o binary from `bytes` at `offset` in `lossy` mode
+    pub fn parse_2(bytes: &'a [u8], mut offset: usize, lossy: bool) -> error::Result<MachO<'a>> {
```

Comment:
definitely don't want this name in the public api; also why is lossy a bool? generally having a bool in a public function is not the best due to boolean blindness.